docs/lago-tutorial/scenarios/jenkins/README.rst

```diff
@@ -0,0 +1,32 @@
+About
```

Comment:
I'll add a small quick guideline, no need to explain the commands, but something like:

```
lago init
lago start
lago status
lago deploy
```

and then explaining what is specific with the env (it sets up a jenkins master with two slaves....)